### DISCUSSION TITLE
Why are u average logits?

### DISCUSSION THREAD

**Discussion by kreininmv** [MIC-DKFZ/nnUNet#2347] [Ideas]
Why are you averaging logits after each fold is released, and not probabilities? It works unstable.
